Gems/Atom/Feature/Common/Assets/Shaders/LightCulling/LightCullingTilePrepare.azsl

```diff
@@ -281,9 +281,19 @@ void MainCS(
 
     float2 minmaxDepth_transparent = ReadTransparentMinMaxMSAA(dispatchThreadID.xy);
 
-    if (o_msaaMode == MsaaMode::Msaa2x)
+    if (o_msaaMode == MsaaMode::None || o_msaaMode == MsaaMode::Msaa2x)
     {
-        float2 opaqueDepthSamples = ReadOpaqueDepthSamples2xMSAA(dispatchThreadID.xy);    
+        float2 opaqueDepthSamples;
+        if (o_msaaMode == MsaaMode::None)
+        {
+            float depth = PassSrg::m_depthBuffer.Load(uint3(dispatchThreadID.xy, 0)).x;
```

Comment:
This is fine for now but really we should be using the super-variant system to do this work. I can look into that as part of my current task.